cve_bin_tool/vex_manager/parse.py

```diff
@@ -1,14 +1,15 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from typing import Any, DefaultDict, Dict, Set, Union
+from collections import DefaultDict
```

Comment:
Looks like this is the line that's causing your tests to fail:

```
   File "/home/runner/work/cve-bin-tool/cve-bin-tool/cve_bin_tool/vex_manager/parse.py", line 4, in <module>
    from collections import DefaultDict
ImportError: cannot import name 'DefaultDict' from 'collections' (/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/collections/__init__.py)
```

Maybe it needs to come from collections.abc?  Not sure off the top of my head but I'm sure you can search for it!

Reply:
Hi, I think it should work now